Kaos-Frontend-Coding-Standards.md

```diff
@@ -136,6 +136,21 @@ At Sonatype we value the stability and maintainability of the code base while st
     }
   }
 ```
+* We denote future bindable members at the top of Angular controllers
```

Comment:
Adding the 'why' to this would help people understand the value in the future.  I see that was captured in the entry on Jasmine Development.